Makefile

```diff
@@ -1,10 +1,13 @@
+Q := @
```

Comment:
What is `Q`?

Reply:
It is a switch to decide how 'verbose' you want to be. Basically it set the `@` in front of a command. I am doing this from forever, not sure where I stole the idea.

- you prepend all commands with `$(Q) <actual command>
- by default (`Q := @`) you silence all commands output
- if you want you can do `Q :=` and all actual command invocations get printed.

Reply:
@morrisonlevi are you 👍  with my answer to this comment?

Reply:
👍